keyboards/the_royal/liminal/keymaps/default/keymap.c

```diff
@@ -0,0 +1,157 @@
+   * ├─────┼────┴┬───┴┬───┴┬───┴┬───┴┬───┴┬───┴┬───┴┬───┴┬───┴┬───┴┬───┴──────┤
+   * │  |  │CAPS │    │    │    │    │    │    │    │    │    │    │    +     │
+   * ├─────┼─────┴─┬──┴─┬──┴─┬──┴─┬──┴─┬──┴─┬──┴─┬──┴─┬──┴─┬──┴─┬──┴─┬────────┤
+   * │     │       │    │    │    │    │    │    │    │    │ [  │ ]  │        │
+   * ├─────┼────┬──┴─┬──┴─┬──┴────┴────┴────┴────┴────┴───┬┴────┴┬───┴─┬──────┤
+   * │     │    │    │    │                               │      │     │      │
+   * └─────┴────┴────┴────┴───────────────────────────────┴──────┴─────┴──────┘
+   */
+  [_RAISE] = LAYOUT_base_kit_all(
+    KC_BSLS,  _______,  _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,   KC_EQUAL,   _______,
+    KC_PIPE, KC_CAPS,   _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,   _______,    KC_PLUS,
+    _______,  _______,  _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    KC_LBRC,   KC_RBRC,    _______,
+    _______,  _______,  _______,    _______,                      _______,                _______,                      _______,    _______,   _______,    _______
+  ),
+
+  /* OTHER layer
+   * ┌─────┬────┬────┬────┬────┬────┬────┬────┬────┬────┬────┬────┬────┬──────┐
+   * │HOME │    │    │    │    │    │    │    │    │    │    │ UP │    │      │
+   * ├─────┼────┴┬───┴┬───┴┬───┴┬───┴┬───┴┬───┴┬───┴┬───┴┬───┴┬───┴┬───┴──────┤
+   * │END  │     │    │    │    │    │    │    │    │    │LEFT│RGHT│          │
+   * ├─────┼─────┴─┬──┴─┬──┴─┬──┴─┬──┴─┬──┴─┬──┴─┬──┴─┬──┴─┬──┴─┬──┴─┬────────┤
+   * │     │       │    │    │    │    │    │    │    │    │DOWN│    │        │
+   * ├─────┼────┬──┴─┬──┴─┬──┴────┴────┴────┴────┴────┴───┬┴────┴┬───┴─┬──────┤
+   * │RESET│    │    │    │                               │      │     │      │
+   * └─────┴────┴────┴────┴───────────────────────────────┴──────┴─────┴──────┘
+   */
+  [_OTHER] = LAYOUT_base_kit_all(
+    _______,    _______,  _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    KC_UP,     _______,   _______,
+    _______,    _______,  _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    KC_LEFT,    KC_RIGHT,  _______,   _______,
+    _______,    _______,  _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,   KC_DOWN,    _______,   _______,
+    TD(TD_RST), _______,  _______,    _______,                      _______,               _______,                       _______,   _______,    _______,   _______
+  )
+};
+
+uint16_t get_tapping_term(uint16_t keycode) {
+  switch (keycode) {
+    case LT(_LOWER, KC_SPACE):
+      return 160;
+    case TD(TD_DBQT):
+      return 230;
+    default:
+      return 200;
```

Comment:
I'd be more comfortable with something like:
```suggestion
#ifndef TAPPING_TERM
      return 200;
#else 
      return TAPPING_TERM;
#endif
```

Reply:
ahhhhh so this is how you do that. I was confused yesterday.